src/components/Attachments/AttachmentView/index.tsx

```diff
@@ -222,15 +223,20 @@ function AttachmentView({
     if (isFileImage) {
         if (imageError && (typeof fallbackSource === 'number' || typeof fallbackSource === 'function')) {
             return (
-                <Icon
-                    src={fallbackSource}
-                    height={variables.defaultAvatarPreviewSize}
-                    width={variables.defaultAvatarPreviewSize}
-                    additionalStyles={[styles.alignItemsCenter, styles.justifyContentCenter, styles.flex1]}
-                    fill={theme.border}
-                />
+                <View style={[styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                    <Icon
+                        src={fallbackSource}
+                        width={variables.iconSizeSuperLarge}
+                        height={variables.iconSizeSuperLarge}
+                        fill={theme.border}
```

Comment:
Consider making this theme.icons instead

Reply:
Ok, thx for the feedback

I tried theme.icon but it"s still light as you can see in the pic below
![Capture d’écran de 2024-10-18 20-50-47](https://github.com/user-attachments/assets/6ed60468-00ff-4709-a7be-fc609c16fb5d)

but a theme that seems to match better the "Attachment not found" text in both light and dark modes is theme.iconHovered as you can see in the pic below

![Capture d’écran de 2024-10-18 20-57-58](https://github.com/user-attachments/assets/d3bf905b-e88a-46b0-8fa6-c78f4cfb85e7)

What do you think ? also @dannymcclain

Reply:
The first pic (`theme.icon`) looks correct!
![image](https://github.com/user-attachments/assets/2843d85a-f3bf-4c9c-a76c-728654812562)

Reply:
Ok @dannymcclain thx for the feedback, @shawnborton wha's your decision ?

Reply:
Yup, Danny is correct!

Reply:
@shawnborton yes, i have updated the code already. thx for the feedback :+1:

Reply:
Thanks!